host/sgx/elf.c

```diff
@@ -1909,7 +1909,8 @@ oe_result_t elf64_load_relocations(
     for (; p != end; p++)
     {
         uint64_t reloc_type = ELF64_R_TYPE(p->r_info);
-        if (reloc_type != R_X86_64_RELATIVE && reloc_type != R_X86_64_TPOFF64)
+        if (reloc_type != R_X86_64_RELATIVE && reloc_type != R_X86_64_TPOFF64
```

Comment:
Can you add a comment regarding the minimal support for `R_X86_64_GLOB_DAT` here? For example:
```
/* The enclave doesn't perform relocations on R_X86_64_GLOB_DAT, but we allow 
 * it for code that checks for the existence of weak symbols before using them. */
```

Reply:
thanks for the suggestion. added in the code.